build/build-sdk-images/tool/base/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN git clone --recurse-submodules -b $GRPC_RELEASE_TAG --depth 1 --shallow-subm
     cd /var/local/git/grpc && \
     mkdir -p cmake/build && \
     cd cmake/build && \
-    cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF ../.. && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF ../.. && \
```

Comment:
Out of curiosity, what does this do?

Reply:
This change triggered the agones-sdk-base image build with latest grpc-version.
Without this, the CI used cache of old image and failed to build grpc codes.

https://console.cloud.google.com/cloud-build/builds/d07c8b4b-a4e7-456e-85cb-e8c2ce5d5870;step=16?project=agones-images

Reply:
Oh got it!

You could have also updated

> Base images for SDKs Bump: 2

To 

> Base images for SDKs Bump: 3

Just to update the file, which resets the build, but no reason to fix I don't think.

Reply:
Oh I didn't know `Base images for SDKs Bump: XX`!!!
After another review, if I should revert `-DCMAKE_BUILD_TYPE=Release`, please let me know.